analysis_plan.md

```diff
@@ -15,18 +15,17 @@
 We will be meeting regularly on Sunday afternoons to discuss general progress and each other's ideas
 
 ## Exploring outlier detection
-The basic DVARS function seems limited because of all the issues discussed in the "ontheproject.md" file. We tried a generalized version of DVARS (dvars_all in metrics.py called via find_outliers), which compares each volumes against all of the others (not just the exact previous one) and generates a metric of how different a given volume is from the "center" of the volumes (the values can be tentatively found in a file called "potential_outliers.txt" in the main folder). This metric is computationally very complex and the computation takes a long time to run. We need to come up with a more efficient way?
-This metric does not have a clear/intuitive interpretation; it flags out a volume as "different", but we do not know why or how that volume differs.  
+The basic DVARS function seems limited because of all the issues discussed in the "ontheproject.md" file. We tried a generalized version of DVARS (dvars_all in metrics.py called via find_outliers), which compares each volumes against all of the others (not just the exact previous one) and generates a metric of how different a given volume is from the "center" of the volumes (the values can be tentatively found in a file called "potential_outliers.txt" in the main folder). This metric is computationally very complex and the computation takes a long time to run (approx 30 minutes!). We need to come up with a more efficient way? This metric does not have a clear/intuitive interpretation; it flags out a volume as "different", but we do not know why or how that volume differs.  
```

Comment:
I guess you could try something simpler.  For example, you take the mean volume and compare against that.  There are various problems with that, of which two obvious ones are that the mean includes outliers, and the person may move slowly during the scan session, so first and last volumes will look particularly different.

Reply:
For the problem of bias on the mean estimator you can use median instead, which is unbiased and gives you the same reference you need more robustly.

Reply:
However, if you don't want to deviate much from DVARS, Tom Nichols proposed a variant of DVARS, called standardized DVARS, which can be used to more easily set a more reliable threshold (i.e., that works on new data)